Sources_v3/StreamChat/APIClient/Endpoints/WebSocketConnectEndpoint.swift

```diff
@@ -1,15 +1,15 @@
 //
-// Copyright © 2020 Stream.io Inc. All rights reserved.
+// Copyright © 2021 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
 
 extension Endpoint {
     static func webSocketConnect<UserData: UserExtraData>(
         userId: UserId,
-        name: String?,
-        imageURL: URL?,
-        role: UserRole = .user,
+        name: String? = nil,
+        imageURL: URL? = nil,
+        role: UserRole? = nil,
         extraData: UserData? = nil
     ) -> Endpoint<EmptyResponse> {
```

Comment:
While working on this PR learned that passing `name/imageURL/role/extraData` when connecting to the web-socket is **not required** and is just a **shorthand** for updating the user data before the connection is established OR creating a new user with the provided values, not the default (nil?) ones.

Reply:
it saves an `updateUser` call but at what cost... We can remove these params from this func

Reply:
Yeah, was gonna preserve passing user data with the request but has a word with Vojta and we decided to connect with just a token. Maybe @VojtaStavik has smth to add here

Reply:
Yes please, let's remove everything but the user id.